.github/workflows/provisioning-tests.yml

```diff
@@ -1,15 +1,18 @@
 name: Provisioning tests
 on:
-  pull_request:
-    branches:
-      - main
-      - release/v*
-    paths-ignore:
-      # omit tests from triggering CI except when CI tests are changed
-      - 'tests/v2/validation/**'
-      - 'tests/v2/actions/**'
-      - 'tests/v2/codecoverage/**'
-      - 'tests/validation/**'
```

Comment:
Should this be added back to allow the provisioning tests to be triggered on PR?